### ISSUE TITLE
Rename the project

### ISSUE THREAD

**Issue by aloisdg** [aloisdg/Ascolorii#6]
"ascolorii" is an obvious fusion of "ascii" and "color". I would love to change it for something shorter and sweeter.

Idea:
- asco
- ascoli
- ascolii
- ascori
- ascorii
- ...

**Comment by Masacaai** [comment]
How about colorscii? I feel like it's a perfect fusion of both words.

**Comment by aloisdg** [comment]
I like your submission actually.  I am not so sure that it is important to rename the project now.

**Comment by Masacaai** [comment]
Yeah, I found your repo while hunting for colorscripts. Cool work tho!
